src/i18n/locales/fr.json

```diff
@@ -0,0 +1,35 @@
+{
+"components.hero.learn": "Apprendre Node",
+  "components.installTabs.readDocs": "Lire la documentation",
+  "components.pagination.previous": "← Précédent",
+  "components.pagination.next": "Suivant →",
+  "components.randomContributor.thankYou": "Nous remercions, {contributor}  pour avoir été un contributeur Node.js, avec {amount} contributions",
+  "components.recentPosts.title": "Messages récents",
+  "components.searchBar.placeholder": "Recherche",
+  "components.searchBar.search.title": "Commencez à taper pour rechercher",
+  "components.searchBar.search.noResults": "Aucun résultat ne correspond à votre requête.",
+  "components.sideBar.title": "Menu",
+  "components.sideBar.items.about": "À propos de",
+  "components.sideBar.items.governance": "Gouvernance de projet",
+  "components.sideBar.items.community": "Communauté",
+  "components.sideBar.items.workingGroups": "Groupes de travail",
+  "components.sideBar.items.releases": "Sorties",
+  "components.sideBar.items.resources": "Ressources",
+  "components.sideBar.items.privacy": "Politique de confidentialité",
+  "components.sideBar.items.security": "Rapports de sécurité",
+  "components.sideBar.items.packageManager": "Package Manager",
+  "containers.authorList.title": "Contributeurs",
+  "containers.navigation.title": "Menu",
+  "pages.download.description": "Téléchargez le code source de Node.js, un programme d'installation préconstruit pour votre plateforme, ou installez via un fichier de configuration.",
+  "pages.download.packageManager": "package manager",
+  "pages.notFound.title": "PAGE NON TROUVÉE",
+  "pages.notFound.description": "La page à laquelle vous essayez d'accéder n'existe pas. Retournez à la page d'accueil ou trouvez ce que vous cherchez dans le menu.",
+  "pages.notFound.takeMeBack": "Ramenez-moi à la page d'accueil",
+  "pages.blog.noEntries": "Aucun article de blog dans cette catégorie.",
+  "pages.docs.apiVersion.select": "Sélectionnez la version de l'API",
+  "pages.index.features.javascript.title": "JavaScript",
+  "pages.index.features.javascript.description": "Node.js fournit un support pour le langage de programmation JavaScript.",
+  "pages.index.features.openSource.title": "Open Source",
+  "pages.index.features.openSource.description": "Node.js est open source et activement maintenu par des contributeurs du monde entier",
+  "pages.index.features.everywhere.description": "Node.js a été adapté pour fonctionner dans une grande variété d'endroits"
```

Comment:
There are missing entries here, check the `en.json` file 🙂

Reply:
Ok I check